### PR TITLE
Ux improvements

### PR DESCRIPTION
**Pull request by gereon77**
fixed part of #89

### CODE REVIEW

File: agot-bg-game-server/src/common/ingame-game-state/westeros-game-state/clash-of-kings-game-state/ClashOfKingsGameState.ts [hunk 1]
```diff
@@ -36,6 +41,10 @@ export default class ClashOfKingsGameState extends GameState<WesterosGameState,
 
     onServerMessage(message: ServerMessage) {
         this.childGameState.onServerMessage(message);
```
Comment (Longwelwind) [inline agot-bg-game-server/src/common/ingame-game-state/westeros-game-state/clash-of-kings-game-state/ClashOfKingsGameState.ts]: This line should be put in the `else` of the condition after, like it is done in other `GameState` classes.

File: agot-bg-game-server/src/messages/ServerMessage.ts [hunk 2]
```diff
@@ -207,3 +207,9 @@ interface ChangeValyrianSteelBladeUse {
     type: "change-valyrian-steel-blade-use";
     used: boolean;
 }
+
+interface NextTrack {
```
Comment (Longwelwind) [inline agot-bg-game-server/src/messages/ServerMessage.ts]: A slightly more descriptive name would be better, like `BiddingNextTrack`.

File: agot-bg-game-server/src/client/game-state-panel/ResolveSingleMarchOrderComponent.tsx [hunk 3]
```diff
@@ -196,6 +196,12 @@ export default class ResolveSingleMarchOrderComponent extends Component<GameStat
             return;
         }
 
+        if(this.plannedMoves.size == 0) {
+            if(!confirm("Do you want to remove your march order?")) {
```
Comment (Longwelwind) [inline agot-bg-game-server/src/client/game-state-panel/ResolveSingleMarchOrderComponent.tsx]: I'm really not a fan of `confirm` prompt but I guess we can keep them for the time being.

In a later PR, we can replace them with native prompts.

File: agot-bg-game-server/src/client/GameLogListComponent.tsx [hunk 4]
```diff
@@ -464,6 +464,16 @@ class GameLogListComponent extends Component<GameLogListComponentProps> {
                 return <>
                     <strong>Wildling strength</strong> reached <strong>{data.wildlingStrength}</strong>, triggering a <strong>Wildling attack</strong>
                 </>;
+            case "march-order-removed":
+                house = this.game.houses.get(data.house);
+                // A "null" for "attacked" means it was an attack against a neutral force
```
Comment (Longwelwind) [inline agot-bg-game-server/src/client/GameLogListComponent.tsx]: This comment is not necessary, I think.

**Comment by Longwelwind** [comment]
On top of the remarks I've made, please try to follow the conventions for commit messages by beginning with an upper-case.

Once it's done, I'll merge!

**Comment by gereon77** [comment]
Provided a new PR #119 .

**Pull request closed without merging**